cli/src/main/scala/com/salesforce/op/cli/CommandParser.scala

```diff
@@ -47,9 +47,12 @@ trait OpCli {
     else success
   }
 
+  private[cli] def tweak(file: File): File =
+    new File(file.getPath.replace("`pwd`/", ""))
```

Comment:
why do we need to hack around `pwd`?

Reply:
Just to make developer's life easier on certain occasions.

Reply:
Makes life easier on some occasions. When one copy-pastes the path, for instance.